Add confirmation option when disconnecting as a follower in Remote Access (#18044)
nvaccess/nvda: user_docs/en/userGuide.md

When checked, NVDA will produce natural-sounding audio cues for Remote Access events.
When unchecked, NVDA will beep for Remote events.

##### Confirm when disconnecting as a follower {#RemoteConfirmDisconnect}

> Use this option to require confirmation before disconnecting from a Remote Access session as a follower.

Reviewer: ```suggestion
Use this option to require confirmation before disconnecting from a Remote Access session when connected as the controlled computer.
```

> Use this option to require confirmation before disconnecting from a Remote Access session when connected as the controlled computer.

Reviewer: ```suggestion
This option controls whether confirmation is required before disconnecting from a Remote Access session when connected as the controlled computer.
```
